addons/crm/tests/common.py

```diff
@@ -242,6 +242,10 @@ def setUpClass(cls):
             'res_id': cls.activity_type_1.id,
         })
 
+    def setUp(self):
```

Comment:
Commit message about the flush seems a bit strange in terms of wording.
Maybe reword a bit and explain exactly why?

> We also specifically add some tracking flush in test performance tests

typo?

Reply:
Typo indeed. Commit message updated.